ambry-api/src/test/java/com.github.ambry/rest/MockBlobStorageServiceFactory.java

```diff
@@ -31,7 +31,7 @@
   private final Router router;
 
   public MockBlobStorageServiceFactory(VerifiableProperties verifiableProperties, ClusterMap clusterMap,
-      RestResponseHandler restResponseHandler, Router router, Notifier notifier) {
+      RestResponseHandler restResponseHandler, Router router, AccountService accountService) {
     this.verifiableProperties = verifiableProperties;
```

Comment:
So, the `accountService` is temporarily not used in this PR?   (also for `clusterMap`)

Reply:
Yeah - none of the tests needed it. If necessary, can be used in the future